de/PCA.py

```diff
@@ -0,0 +1,23 @@
+""" Performs PCA analysis on RNAseq knockout data using functions from PCA_helpers.py"""
+from .PCA_helpers import *
```

Comment:
It's best to avoid wildcard imports. Rather than `*`, list the function you want to import.